modules/weko-records/weko_records/models.py

```diff
@@ -546,7 +546,6 @@ def __iter__(self):
                 if isinstance(value, str):
                     yield (name, value)
 
-
```

Comment:
Modify source code/ files not in the ticket scope

Reply:
Thank for your comment.
It had been fix at last rev.